src/modules/auth/webauthn/WebAuthnScreen.js

```diff
@@ -66,6 +79,7 @@ function WebAuthnCard(props) {
 			setIsLoading(false);
 			return;
 		}
+		setGlobalEditMode(false);
```

Comment:
@Aringoaway This will not work when you fail to get authenticators, since it will be returned in `catch` exception. Please fix that.